todo_app/Ansible/my-ansible-playbook.yml

```
@@ -0,0 +1,78 @@
+- name: Install todo app on new web servers
+  hosts: webservers
+  remote_user: ec2-user
+
+  vars_prompt:
+    - name: api_key
+      prompt: What is the Trello API key?
+
+    - name: token
+      prompt: What is the Trello Token?
+
+    - name: board_id
+      prompt: What is the Trello board id?
+
+    - name: todo_list_id
+      prompt: What is the to do list id?
+
+    - name: started_list_id
+      prompt: What is the started list id?
+
+    - name: done_list_id
+      prompt: What is the done list id?
+
+  tasks:
+  - name: Install git
+    yum:
+      name: git
+      state: latest
+    become: yes
+
+  - name: Install Python3
+    yum:
+      name: python3
+      state: latest
+    become: yes
```

Minor, but if we wanted to we could combine these:
```suggestion
  - name: Install git
    yum:
      name: 
        - git
        - python3
      state: latest
    become: yes
```